plugin.php

```diff
@@ -19,6 +19,8 @@
 }
 
 require_once 'inc/namespace.php';
+require_once 'inc/audiences/namespace.php';
+require_once 'inc/audiences/rest_api/namespace.php';
 require_once 'inc/utils/namespace.php';
```

Comment:
These (and the existing requires too) should be full absolute path references, otherwise they can be hijacked by the server's include_path.
```suggestion
require_once __DIR__ . '/inc/namespace.php';
require_once __DIR__ . '/inc/audiences/namespace.php';
require_once __DIR__ . '/inc/audiences/rest_api/namespace.php';
require_once __DIR__ . '/inc/utils/namespace.php';
```